<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_DlgOptionsMakerCam</class>
 <widget class="QDialog" name="QG_DlgOptionsMakerCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>522</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>522</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export as CAM/plain SVG</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="gbLayers">
     <property name="title">
      <string>Layers</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout" name="hLayoutDefaultElementWidth">
          <item>
           <widget class="QLabel" name="lbDefaultElementWidth">
            <property name="text">
             <string>Default width of elements, mm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dSpinBoxDefaultElementWidth">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkInvisibleLayers">
          <property name="text">
           <string>Export invisible layers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkConstructionLayers">
          <property name="text">
           <string>Export visible construction layers</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="gbBlocks">
     <property name="title">
      <string>Blocks</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="checkBlocksInline">
          <property name="text">
           <string>Convert blocks to safe inline SVG content</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="gbEllipses">
     <property name="title">
      <string>Ellipses / Ellipse arcs</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="checkEllipsesToBeziers">
          <property name="text">
           <string>Approximate ellipses and ellipse arcs with cubic b√©ziers</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="gbImages">
     <property name="title">
      <string>Images</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="checkImages">
          <property name="text">
           <string>Raster Image export</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="gbDashLines">
     <property name="title">
      <string>Dash/Dot Lines</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout" name="hLayoutDashLinePatternLength">
          <item>
           <widget class="QLabel" name="lbDashLinePatternLength">
            <property name="text">
             <string>Length of the pattern, mm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dSpinBoxDashLinePatternLength">
            <property name="maximum">
             <double>999.990000000000009</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkDashDotLines">
          <property name="text">
           <string>Bake dash/dot lines to SVG path</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QG_DlgOptionsMakerCam</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QG_DlgOptionsMakerCam</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
